proxy/tracing_test.go

```diff
@@ -167,9 +167,7 @@ func (t *tracer) Extract(format interface{}, carrier interface{}) (ot.SpanContex
 }
 
 // SpanContext interface
-func (s *span) ForeachBaggageItem(handler func(k, v string) bool) {
-	return
-}
+func (s *span) ForeachBaggageItem(_ func(k, v string) bool) {}
```

Comment:
Just because you're at it, we could also _ the receiver `s`, like:

```Go
func (_ *span)  ForeachBaggageItem(_ func(k, v string) bool) {}
```

Reply:
Yes, thanks, done so.